src/routes/docs/tooling/keyboard/+page.markdoc

```diff
@@ -0,0 +1,61 @@
+
+# Shortcuts {% #shortcuts %}
+
+Only in the Appwrite console can you use these keyboard shortcuts to access Appwrite's console. Appwrite's keyboard shortcuts follow the following pattern: use the first letters from the call to action and the resource, product, service, or page.  For example, using the shortcut `G,` then `S` goes to storage, but the shortcut `G` then `E` is if you want to go to settings, because `G` then `S` is already taken, so the following letter of settings.
+
+## Global shortcuts {% #global-shortcuts %}
+
+Appwrite users can utilize global shortcuts anywhere in the Appwrite console, allowing instant access to these features, whether on their account page or creating an organization page.
+
+|PC (Windows)|Mac (OS)|Action|
+|----|----|----|
+|`Ctrl` + `K`|`⌘` + `K`|Access Command Center|
+
+
+|Shortcut|Action|
+|----|----|
+|Set theme to light|`T` then `L`|
+|Set theme to dark|`T` then `D`|
+|Set theme to dark|`T` then `A`|
+|Ask the AI|`A` then `I`|
+
+## Project shortcuts {% #project-shortcuts %}
+
+Appwrite users can utilize project shortcuts once they create a project. These shortcuts allow them to access the project's database, storage, messaging, auth, settings, and overview.
+
+|Shortcut|Action|
+|----|----|
+|Go to Auth|`G` then `A`|
+|Go to Storage|`G` then `S`|
+|Go to Databases|`G` then `D`|
+|Go to Functions|`G` then `F`|
+|Go to Projects|`G` then `P`|
+|Go to Overview|`G` then `O`|
+|Go to Messaging|`G` then `M`|
+|Go to Settings|`G` then `E`|
+|Create Organization|`C` then `O`|
+
+# The Appwriter {% #the-appwriter %}
+
+{% only_dark %}
+![The Appwriter](/images/docs/keyboard/dark/appwriter.png)
+{% /only_dark %}
+
+{% only_light %}
+![The Appwriter](/images/docs/keyboard/appwriter.png)
+{% /only_light %}
+
+The Appwriter assigns Appwrite icons from the Appwrite Console and Docs to specific keys, making memorizing keyboard shortcuts easier. For example, `G` then `D` is the shortcut for going to databases, and the icon on the keyboard matches the icon in Appwrite Databases. To win an Appwriter, attend Appwrite events by following us and look for Appwrite's next giveaway on the following social media: [Discord](https://discord.com/invite/GSeTUeA), [Github](https://github.com/appwrite/appwrite), [X](https://x.com/intent/follow?screen_name=appwrite) and [YouTube](https://www.youtube.com/c/appwrite).
```

Comment:
Lets move the how you get it part to the end as a new paragraph so we have proper structure. 

We can also start it with something like this:
There are multiple ways to get an Appwriter. You can either win it by...

and end it by saying the keyboard will also be available for purchasing on the Appwrite Store (link) by the end of 2024